core/block.js

```diff
@@ -27,7 +27,6 @@
 goog.provide('Blockly.Block');
 
 goog.require('Blockly.Blocks');
-goog.require('Blockly.Comment');
```

Comment:
It's okay to remove it from here, but somewhere in Blockly we need to include it, otherwise compressed won't have it. It looks like block_svg is the appropriate place to place that, as it instantiates a comment in ``setCommentText``.